ga-ccd-definition/CaseField/CaseFieldGAspec-CUI-nonprod.json

```diff
@@ -86,5 +86,13 @@
     "FieldTypeParameter": "TranslatedDocument",
     "SecurityClassification": "Public",
     "Searchable": "N"
+  },
+  {
+    "CaseTypeID": "GENERALAPPLICATION",
+    "ID": "coSCApplicationStatus",
+    "Label": "CoSC Application Status",
+    "FieldType": "YesOrNo",
```

Comment:
should be fixed list - active, processed, inactive

Reply:
I removed because this fields should be on main case.